python/tk_multi_loader/model_latestpublish.py

```diff
@@ -326,7 +326,7 @@ def _load_external_data(self):
         for tree_view_item in self._treeview_folder_items:
 
             # compute and store a hash for the tree view item so that we can access it later
-            tree_view_item_hash = str(hash(tree_view_item))
+            tree_view_item_id = id(tree_view_item)
```

Comment:
This was interesting. Python 3 does not allow hashing of any data type. Fortunately, a much simpler approach is to take the `id` of the widget.

Reply:
I think this looks fine, as long as this id doesn't get cached anywhere so that it outlives the session. Reading the method description, it seems that it doesn't, so should be good.